www/src/components/sidebar/item-link.js

```diff
@@ -99,7 +99,7 @@ export default function ItemLink({ item, overrideCSS }) {
         onClick={onLinkClick}
         to={item.link}
       >
-        {item.ui === `steps` && (
+        {isSteps && (
           <span
             sx={{
               bg: `white`,
```

Comment:
Hey @Pyrax! 👋 
Thanks so much for taking care of this! 🙏 

Could you change the `bg` to `ui.border` as mentioned in https://github.com/gatsbyjs/gatsby/pull/24129#issuecomment-630103040?

```suggestion
              bg: `ui.border`,
```